ledger/caching/src/main/scala/com/daml/caching/MappedCache.scala

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.caching
+
+private[caching] final class MappedCache[Key, Value, NewValue](
+    from: Value => NewValue,
```

Comment:
Maybe `fromDelegate` to be more specific? You have to read through the implementation to understand the direction (i.e., _from_ goes from where?).

Reply:
I've changed them to "mapAfterReading" and "mapBeforeWriting".